### PR TITLE
fix: Silence errors and usage

### PR DESCRIPTION
**Pull request by scuzhanglei**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -309,7 +309,9 @@ func main() {
 	cmdVersion.PersistentFlags().StringVarP(&versionOutput, "output", "o", versionOutput, "Output format; available options are 'json'")
 
 	rootCmd := &cobra.Command{
-		Use: "knest",
+		Use:           "knest",
+		SilenceUsage:  true,
+		SilenceErrors: true,
```
Comment (fengye87) [inline main.go:314]: We should still print errors?

Comment (scuzhanglei) [reply main.go:314]: for example when knest call kubectl command, if kubectl return an error, kubectl will print it, and knest will print it again, with `SilenceErrors` only print the error once.

```
Error from server (NotFound): clusters.cluster.x-k8s.io "abc" not found
delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
exit status 1
```

```
Error from server (NotFound): clusters.cluster.x-k8s.io "abc" not found
Error: delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
exit status 1
```

Comment (fengye87) [reply main.go:314]: What about errors from knest?

Comment (scuzhanglei) [reply main.go:314]: the repeated print are caused by https://github.com/smartxworks/knest/blob/main/main.go#L322, maybe we should keep `SilenceErrors=false`, and remove https://github.com/smartxworks/knest/blob/main/main.go#L322

Comment (scuzhanglei) [reply main.go:314]: example logs:
```
Error from server (NotFound): clusters.cluster.x-k8s.io "abc" not found
Error: delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
exit status 1
```

```
Error from server (NotFound): clusters.cluster.x-k8s.io "abc" not found
Error: delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
exit status 1
```

Comment (fengye87) [reply main.go:314]: 👌Let's do that

**Review by fengye87 — APPROVED** [review]

**Review by fengye87 — APPROVED** [review]

**Pull request merged**